lib/CXGN/BrAPI/v1/Studies.pm

```diff
@@ -777,11 +777,12 @@ sub studies_table {
 		#print STDERR Dumper \@trait_names;
 		my @header_ids;
 		foreach my $t (@trait_names) {
-            if ($t eq 'notes'){
-                push @header_ids, 0;
-            } else {
-                push @header_ids, SGN::Model::Cvterm->get_cvterm_row_from_trait_name($self->bcs_schema, $t)->cvterm_id();
-            }
+			if ($t eq 'notes'){
+				push @header_ids, 0;
+			} else {
+				next unless $t =~ /\|/;
```

Comment:
$t =~ /\|/; may not always be true even for valid traits (such as when trait abbreviations are used in a file) ???